CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 The change log describes what is "Added", "Removed", "Changed" or "Fixed" between each release.
 
+# 1.20.0 - to be released
+
+- Support PHP 8.
+- Support Guzzle 7.
+- BC BREAK: Remove support for Guzzle 6.
```

Comment:
Let me know if you agree with this.

Reply:
looking at the configuration files, we seem to still support guzzle 5 - if that still works, i suggest we don't replace guzzle 6 with 7, but just add 7. or is that not possible?

Reply:
Yes, that’s definitely possible. Alle Guzzles >5 allow PHP 8 (they have `php: >=5.4.0` or something similar).

While it’s easiest if PHP 8 users just switch to the latest Guzzle’s HTTPlug adapter, I agree that it may ease maintenance if we don’t introduce a BC break here. So let me revert this change and see if this bundle can continue to support Guzzle 6.

Reply:
i would expect guzzle 6 to not work with php 8, but it should continue to work on older versions. our guzzle6-adapter is not installable in php 8 anyways.